### PR TITLE
Ceizak NM - Unfettered Twitherym - behavior, hp, mp, mob skills.

### PR DESCRIPTION
**Pull request by Yuchemey**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

This PR introduces mechanics to the Unfettered Twitherym NM, adding functionality for its mob skills, spells, HP/MP/Job/ItemLevel attributes and more. Specifically, after the mob performs a mob skill (at 74.49% HP), it becomes resistant to physical, magical, breath, and ranged damage. The only way to remove its damage resistance is by performing a skillchain with the correct magic affinity (e.g., fire, water, etc.).

Fire, Wind, Lightning, and Light work with Light.
Other elements work with their respective affinities (e.g., Fire skillchains for fire-based mobs).

## Steps to test these changes

Pop the NM using a Celadon Ygrette I at the appropriate location (J-7).
Engage the Unfettered Twitherym and observe its behavior as its HP reaches 74%, 49% and 24%.
Test the damage resistance by attacking with different types of damage (physical, magical, breath, ranged).
Perform a skillchain with the correct magic affinity (e.g., fire for fire-based, light for light-based).
Verify that the damage resistance is removed after the correct skillchain is executed.

### CODE REVIEW

File: scripts/actions/mobskills/tempestuous_upheaval.lua [hunk 1]
```diff
@@ -0,0 +1,29 @@
+-----------------------------------
+--  Tempestuous Upheaval
+--
+--  Description: Deals Wind damage to enemies within an area of effect. Additional effect: Knockback
+--  Type: Magical (Wind)
+--  Utsusemi/Blink absorb: 2-3 shadows
+--  Range: 10' radial
+--  Notes: The knockback is rather severe.
```
Comment (WinterSolstice8) [inline scripts/actions/mobskills/tempestuous_upheaval.lua:8]: Does the knockback work? If not add a todo comment

File: sql/mob_pools.sql [hunk 2]
```diff
@@ -4915,7 +4915,7 @@ INSERT INTO `mob_pools` VALUES (4858,'Mafic_Spider','Mafic_Spider',235,0x0000390
 INSERT INTO `mob_pools` VALUES (4859,'Felsic_Eruca','Felsic_Eruca',107,0x0000960100000000000000000000000000000000,4,4,12,240,100,0,0,0,0,0,0,0,0,129,12,0,2,0,0,107,107);
 INSERT INTO `mob_pools` VALUES (4860,'Wooden_Ungeweder','Wooden_Ungeweder',99,0x0000660800000000000000000000000000000000,4,5,12,240,100,0,1,0,0,0,658,0,112,1155,0,0,12,0,0,99,99);
 INSERT INTO `mob_pools` VALUES (4861,'Jungle_Baelfyr','Jungle_Baelfyr',102,0x0000650800000000000000000000000000000000,4,5,12,240,100,0,1,0,0,0,36,0,1934,129,0,0,17,0,0,102,102);
-INSERT INTO `mob_pools` VALUES (4862,'Unfettered_Twitherym','Unfettered_Twitherym',338,0x0000E70900000000000000000000000000000000,1,1,12,240,100,0,1,0,1,0,0,64,7,1159,5,0,0,4,0,338,338);
+INSERT INTO `mob_pools` VALUES (4862,'Unfettered_Twitherym','Unfettered_Twitherym',338,0x0000E70900000000000000000000000000000000,110,110,12,240,100,0,1,0,1,0,0,64,7,1159,5,0,508,4,0,338,338);
```
Comment (UmeboshiXI) [inline sql/mob_pools.sql:4918]: Job/Subjob ID is out of range.

**Comment by WinterSolstice8** [comment]
The Dealer Moogle error in LLS is not you, please ignore it

File: sql/mob_pools.sql (hunk 2 above)
Comment (Yuchemey) [reply sql/mob_pools.sql:4918]: it is sorry thought i fixed it lol

File: scripts/actions/mobskills/tempestuous_upheaval.lua (hunk 1 above)
Comment (Yuchemey) [reply scripts/actions/mobskills/tempestuous_upheaval.lua:8]: it does

Comment (WinterSolstice8) [reply scripts/actions/mobskills/tempestuous_upheaval.lua:8]: Nice

**Comment by WinterSolstice8** [comment]
Going to have to amend your commit title for CI to run "Update File" is not descriptive

**Comment by WinterSolstice8** [comment]
I saw you commented on your commit, that could happen if you had merged the base branch with your commit. If you use rebase that should never happen 

Once the PR is good to go I can clean up the commits for you so I wouldn't worry about it for now

In the future just be sure to use rebase

**Comment by Yuchemey** [comment]
> I saw you commented on your commit, that could happen if you had merged the base branch with your commit. If you use rebase that should never happen
> 
> Once the PR is good to go I can clean up the commits for you so I wouldn't worry about it for now
> 
> In the future just be sure to use rebase

thank you.

**Review by WinterSolstice8 — APPROVED** [review]

**Comment by WinterSolstice8** [comment]
cleaned up the commits (you might need to use `git pull --force` locally)

**Comment by Yuchemey** [comment]
> cleaned up the commits (you might need to use `git pull --force` locally)

I kinda have a problem. Everything is marked checked and i cannot amend my last commit.
I'll have to re-do it tomorrow.

**Comment by WinterSolstice8** [comment]
What's the error message? everything is fixable

**Comment by Yuchemey** [comment]
> What's the error message? everything is fixable

it doesnt say an error. when i first PR'ed ! became "checked" so i try to amend my last commit, then i merged, when i merged i lost my progress.

**Comment by WinterSolstice8** [comment]
Use the command line, there has to be an error. Checked means nothing to me, git, or anyone. Your GUI is not helping you

**Comment by Yuchemey** [comment]
> Use the command line, there has to be an error. Checked means nothing to me, git, or anyone. Your GUI is not helping you

found  a way to fix it

**Comment by Yuchemey** [comment]
same error as before
"dealer_moogle.lua:3489:49" probably my doing i had to remake my PR from scratch (some what)

**Comment by WinterSolstice8** [comment]
Nah, that's not you, you can ignore it

**Comment by Yuchemey** [comment]
> Nah, that's not you, you can ignore it

I know i can't go any further though

**Comment by WinterSolstice8** [comment]
I have no idea what you're talking about. Ignore that error.

**Comment by Yuchemey** [comment]
> I have no idea what you're talking about. Ignore that error.

While doing the "
build / Linux_LuaLanguageServer (pull_request)" I get an error how do i ignore it?

**Comment by WinterSolstice8** [comment]
You continue on as if it's not there... ignoring it.

**Comment by Yuchemey** [comment]
> You continue on as if it's not there... ignoring it.

So even if i have a fail check,  it's completed?

**Comment by WinterSolstice8** [comment]
This particular error isn't caused by you, so yeah it's fine

**Review by Xaver-DaRed — DISMISSED** [review]

**Comment by Yuchemey** [comment]
Forgot to add unversion files.

File: scripts/actions/mobskills/blackout.lua [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    local power = math.random(10, 20)
+    local duration = math.random(60, 180)
+
+    xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.PARALYSIS, power, 0, duration)
+    xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.SILENCE, power, 0, duration)
```
Comment (Xaver-DaRed) [inline scripts/actions/mobskills/blackout.lua]: Silence power should be 1.

How were this numbers reached? Where was the info gotten from?

Comment (Yuchemey) [reply scripts/actions/mobskills/blackout.lua]: Tested on FFXI Server using the i117 as a PLD/NIN
Tested on my server using the same gear as PLD/NIN with the same stats as in the FFXI server
Fought NMs Over and over about 20 times until i got the approximated results. For mob skills effect, also used 
Bg-wiki as a reference. If there is a way to find out the numbers using capture please do let me know and i will go and do more.

Changed the Silence power.

Comment (Yuchemey) [reply scripts/actions/mobskills/blackout.lua]: Fixed Silence power

Comment (Yuchemey) [reply scripts/actions/mobskills/blackout.lua]: > Tested on FFXI Server using the i117 as a PLD/NIN Tested on my server using the same gear as PLD/NIN with the same stats as in the FFXI server Fought NMs Over and over about 20 times until i got the approximated results. For mob skills effect, also used Bg-wiki as a reference. If there is a way to find out the numbers using capture please do let me know and i will go and do more.

**Review by Xaver-DaRed — APPROVED** [review]

**Pull request merged**